### PR TITLE
macOS Big Sur

### PR DESCRIPTION
**Pull request by fxcoudert**
- Not yet ready for merging, to be shipped as soon as Big Sur ships.
- Related: https://github.com/Homebrew/install/pull/346

### CODE REVIEW

File: Library/Homebrew/os/mac/xcode.rb [hunk 1]
```diff
@@ -17,9 +17,9 @@ module Xcode
       # CI systems have been updated.
       # This may be a beta version for a beta macOS.
       def latest_version
-        latest_stable = "12.1"
+        latest_stable = "12.2"
         case MacOS.version
-        when "11.0"  then "12.1"
+        when "11.0"  then "12.2"
         when "10.15" then latest_stable
```
Comment (jonchang) [inline Library/Homebrew/os/mac/xcode.rb:23]:
```suggestion
        when "11.0"  then latest_stable
        when "10.15" then "12.2"
```

**Comment by MikeMcQuaid** [comment]
> * to be shipped as soon as Big Sur ships.

I don't think this can ship until at least https://github.com/Homebrew/brew/pull/9010 does and/or we have working homebrew-core CI.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
This should not ship until we have working Big Sur CI. We cannot support what we do not have CI for.

**Comment by dvaldivia** [comment]
Will this PR make it so homebrew only works with XCode 12.2 for macOS Big Sur 11.0?

**Comment by fxcoudert** [comment]
We only support the latest Xcode on all platforms

**Comment by dvaldivia** [comment]
@fxcoudert my bad, I though XCode 12.1 was the lastest for macOS 11.0

**Comment by mathieuhendey** [comment]
Now that Xcode 12.2 has been released through the App Store (along with its CLI tools (see image), can this be merged?

Screenshot 2020-11-14 at 08 19 32

**Comment by dvaldivia** [comment]
@mathieuhendey seems they are waiting to set up some CI for Big Sur

**Comment by fxcoudert** [comment]
@mathieuhendey we are setting up a CI and will start supporting Big Sur as soon as we are ready

**Comment by fxcoudert** [comment]
Rebasing. I'm upping the minimum Xcode for Big Sur to 12.2 because 12.0 did not have a Big Sur SDK. 12.2 is the version that coincided with the Big Sur release, as well.

**Comment by fxcoudert** [comment]
I think it is time for this PR, and companion https://github.com/Homebrew/install/pull/346
- We have nodes available for Homebrew core, we have 15% coverage of core bottles in 12 hours.
- There have been no major crisis
- We're likely to turn on Big Sur CI for all formulas in 24 to 48 hours (i.e., around Monday)
- For things that are not bottled, Catalina bottles are used anyway

**Comment by fxcoudert** [comment]
and Big Sur CI is now turned on

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks @fxcoudert!

**Pull request merged**